src/strongbox/ui/cli.clj

```diff
@@ -351,6 +351,32 @@
         addon-id (utils/extract-addon-id addon)]
     (add-tab tab-id (or (:dirname addon) (:label addon) (:name addon) "[bug: missing tab name!]") closable? addon-id)))
 
+(defn-spec addon-has-log-level? boolean?
```

Comment:
docstr, tests

Reply:
wow, logging tests are turning out to be a huge pain. Something very mysterious going on with timbre ...

Reply:
timbre is ... not updating the state during the test but the next test *is* using the updated state, so we have two different atoms in play ... fucking weird.

... urgh, thread-local `binds` https://github.com/ptaoussanis/timbre/blob/master/src/taoensso/timbre.cljc#L223-L224

I wonder if some config setter, or the tests or something is running in a different thread. That might explain this

Reply:
what an ordeal! I have something that is working now but talk about Weirdness. I wasn't able to replicate the problem in a pared down example, in fact it all worked just as expected so the problem was very definitely in strongbox. Not only is there use of `binding` but also `alter-var-root` in timbre when using `with-config` which made me examine my own use of that during testing. Lo and behold, disabling it had it behaving as it should.

But then I still had the problem of the logging ns directly modifying the `timbre/*config*` and the statefulness of the new UI appender during testing. I had a test failing mysteriously when run as part of a ns but not individually.

Like any bit of statefulness I'm attempting to control it more tightly, pushing the logging init into functions called during `start` and the removing the stateful appender during `stop`. A watch has been set to update the appender whenever the selected addon dir changes, rather than commingle that logic with the changing dir logic, etc.